src/mgr/PyModuleRegistry.h

```diff
@@ -227,5 +227,15 @@ class PyModuleRegistry
     return v;
   }
 
+  bool is_module_active(const std::string &name) {
```

Comment:
```suggestion
  bool is_module_active(std::string_view name) {
```

Reply:
The same as above. It needs to be converted back to std::string.